Apps/Database/src/Migrations/20200423173644_Communication.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
```

Comment:
Copyright header and base ignore needs to be added as per other migrations to avoid smells.